tests/std/tests/usual_latest_matrix.lst

```diff
@@ -3,24 +3,24 @@
 
 RUNALL_INCLUDE .\prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest"
+PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest /permissive-"
 RUNALL_CROSSLIST
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0 /Zc:char8_t-"
-PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /Zc:wchar_t- /Zc:preprocessor"
+PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=0 /Zc:wchar_t- /Zc:preprocessor"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=2 /fp:except /Zc:preprocessor"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /await"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /analyze:only"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=0 /fp:strict"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=1"
-PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /await"
+PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /permissive /await"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /analyze:only"
-PM_CL="/Za /MD /permissive-"
-PM_CL="/Za /MDd /permissive-"
-PM_CL="/BE /c /MD"
-PM_CL="/BE /c /MTd /permissive-"
+PM_CL="/Za /MD"
+PM_CL="/Za /MDd"
+PM_CL="/BE /c /MD /permissive"

```

Comment:
This appears to be the only line attempting to preserve `/std:c++latest /permissive` for `/BE`. I thought strict vs. lax had no effect for `/BE` (although I am less sure than for Clang).

Reply:
It does have no effect - these should be `/permissive-` for the reasons I cited earlier.